tools/serverpod_cli/bin/util/command_line_tools.dart

```diff
@@ -82,7 +89,7 @@ class _CommandFormatter {
   late final List<String> args;
 
   _CommandFormatter(String command, this.args) {
-    this.command = Platform.isWindows ? '$command.bat' : command;
```

Comment:
Is this not breaking commands other than the script? E.g. `flutter create`?

Reply:
okie.